common/djangoapps/course_modes/models.py

```diff
@@ -133,13 +133,14 @@ class CourseMode(models.Model):
 
     history = HistoricalRecords()
 
-    HONOR = u'honor'
```

Comment:
I assume we removed the _u_'s because we no longer need them?

Reply:
Yep! `u''` is a Py2.7 thing. Just cleaning up while I'm in here.